src/SessionInstance.php

```diff
@@ -69,7 +75,12 @@ protected function init()
 
         session_name($this->name);
 
-        session_start();
+        if ($this->sessionID) {
+            session_id($this->sessionID);
+        }
+
+        # Start the session up, but ignore the error about headers already being sent
+        @session_start();
```

Comment:
I have amended the commit. If I get time I'll add that `disableCookies` function. Thanks !